zxcvbn-test/ZxcvbnTest.cs

```diff
@@ -87,7 +87,7 @@ public class ZxcvbnTest
         [TestMethod]
         public void RunAllTestPasswords()
         {
-            var zx = new Zxcvbn.Zxcvbn(new Zxcvbn.DefaultMatcherFactory());
+            var zx = new Zxcvbn.PasswordMetric(new Zxcvbn.DefaultMatcherFactory());
```

Comment:
Is the namespace `Zxcvbn` necessary?

Reply:
Addressed in: https://github.com/messerli-informatik-ag/zxcvbn-cs/pull/4/commits/5a055b5c16c4173c813dadf7d1b0f179903377ca

Reply:
Addressed in: https://github.com/messerli-informatik-ag/zxcvbn-cs/pull/4/commits/ad996c0914ef81f3d118fb758225b73a8e0e2213